README.md

```diff
@@ -26,6 +26,10 @@ Before submitting a new entry it is important to review the following guidelines
 > [!NOTE]
 > At the moment the landscape is generated daily, so once your PR is merged your changes should be visible before 24 hours.
 
+## Extras fields
+
+The `extras` section in the landscape.yml file allows you to include various fields to provide essential details about your project. This section helps users understand the project's purpose and find more information. For a detailed list of possible fields please refer to the [Extras Fields Documentation](./docs/extra_fields.md).
```

Comment:
Please note that the section is named `extra` (singular), it appears in plural in a few places in the PR 😇

Reply:
Updated to "extra"